src/ai/susi/mind/SusiSkill.java

```diff
@@ -500,6 +500,7 @@ public static JSONObject getSkillMetadata(String model, String group, String lan
         skillMetadata.put("dynamic_content", false);
         skillMetadata.put("examples", JSONObject.NULL);
         skillMetadata.put("skill_rating", JSONObject.NULL);
+        skillMetadata.put("week_usage", 0);
```

Comment:
Is it necessary to add this to the skill metadata? This is not good in a concurrent environment. Different calls will modify the metadata all the time. Furthermore: naming this "week_usage" is not a generic way. If the usage is sorted in a generic way a field should not be named as such.